starter_code/css/style.css

```diff
@@ -0,0 +1,221 @@
+/******************************************
+/* SETUP
+/*******************************************/
+
+/* Box Model Hack */
+* {
+  -moz-box-sizing: border-box; /* Firexfox */
+  -webkit-box-sizing: border-box; /* Safari/Chrome/iOS/Android */
+  box-sizing: border-box; /* IE */
+}
+
+/* Clear fix hack */
+.clearfix:after {
+  content: ".";
+  display: block;
+  clear: both;
+  visibility: hidden;
+  line-height: 0;
+  height: 0;
+}
+
+.clear {
+  clear: both;
+}
+
+.alignright {
+  float: right;
+  padding: 0 0 10px 10px; /* note the padding around a right floated image */
+}
+
+.alignleft {
+  float: left;
+  padding: 0 10px 10px 0; /* note the padding around a left floated image */
+}
+/* My code*/
+header
+{
+display: flex;
+flex-direction: column;
+flex-wrap: wrap;
+background-color: black;
+height:200px;
+border-bottom: yellow solid 10px;
+}
+.logo
+{
+  color:yellow;
+  font-size: 50px;
+  font-weight: bolder;
+  padding: 30px 40px;
+  font-family: 'Merriweather';
+}
+.MenuBar
+{
+  width: 80%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-self: flex-end;
+  align-items: baseline;
+}
+
+.MenuBar a
+{
+
+  color: white;
+   text-decoration: none;
+
+}
+.MenuBar a:nth-of-type(1)
+{
+  font-family: 'Anton';
+  font-size: 200%;
+}
+.MenuBar a:nth-of-type(2)
+{
+  font-family: 'Anton';
+  font-size: 200%;
+}
+.MenuBar a:nth-of-type(3)
+{
+  font-family: 'open sans';
+  font-size: 150%;
+}
+.MenuBar a:nth-of-type(4)
+{
+  font-family: 'open sans';
+    font-size: 150%;
+}
+.MenuBar a:nth-of-type(5)
+{
+  font-family: 'open sans';
+    font-size: 150%;
+}
+del
+{
+  color: gray;
+}
+ins
+{
+  text-decoration: underline yellow ;
+}
+main
+{
+  display: flex;
+  justify-content: flex-end;
+  background-image: url(../images/coworking.jpg);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 620px;
+  width: auto;
+}
+.Top
+{
+  background-color: gainsboro;
+    height:100px;
+    text-align: center;
+    font-size: 40px;
+
+}
+.bottom
+{
+  background-color: white;
+    height:600px;
+    display: grid;
+  grid-template-columns: 30fr 30fr 30fr;
```

Comment:
You can simply use `1fr` it will divide evenly.
[See more](https://university.webflow.com/article/the-fr-unit#so-how-can-fractions-(fr)-save-the-day)
```suggestion
  grid-template-columns: 1fr 1fr 1fr;
```